code/modules/surgery/surgery_step.dm

```diff
@@ -77,12 +79,13 @@
 	var/implement_speed_mod = 1
 	if(implement_type) //this means it isn't a require hand or any item step.
 		implement_speed_mod = implements[implement_type] / 100.0
-
 	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod) * target.mob_surgery_speed_mod
 	var/modded_time = time * speed_mod
 
-
-	fail_prob = min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
+	if(user == target)
+		fail_prob = min(max(0, modded_time - time),99)
+	else
+		fail_prob = max(5, min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99))//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
```

Comment:
buhh?? why does doing surgery to other people have a 5% chance to fail?

Reply:
because I'm a fucking retard